<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dennis Simpson</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>871</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <widget class="QPushButton" name="tsv_file_select_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select TSV File</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>302</number>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>9</y>
      <width>911</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Gupta Lab Opentrons GUI v0.1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="closeGUI_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>650</y>
      <width>351</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Close GUI</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="simulate_run_btn">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>320</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate Run</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="run_simulation_output">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>390</y>
      <width>921</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Simulation Output</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_program_combobx">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>220</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Select Program to Simulate</string>
    </property>
   </widget>
   <widget class="QLabel" name="select_program_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Program for Simulation:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
